packages/health-check/src/checks/verbose.js

```diff
@@ -1098,69 +1099,60 @@ function parseHeaderString(header) {
 }
 
 // skylinkVerification verifies a skylink against provided information.
-function skylinkVerification(done, { name, skylink, bodyHash, headers, statusCode }, { redirects, method } = {}) {
+async function skylinkVerification(done, expected, { followRedirect = true, method = "get" } = {}) {
   const time = process.hrtime();
 
   // Create the query for the skylink
-  const query = `${process.env.PORTAL_URL}/${skylink}`;
-
-  // Get the Skylink
-  superagent[method || "get"](query)
-    .set("cookie", "nocache=true")
-    .redirects(redirects)
-    .ok((res) => (redirects === undefined ? res.ok : res.status < 400))
-    .responseType("blob")
-    .then(
-      (response) => {
-        const entry = { name, up: true, statusCode: response.statusCode, time: calculateElapsedTime(time) };
-        const info = {};
-
-        if (statusCode && statusCode !== response.statusCode) {
-          entry.up = false;
-          info.statusCode = { expected: statusCode, current: response.statusCode };
-        }
+  const query = `${process.env.PORTAL_URL}/${expected.skylink}`;
 
-        // Check if the response body is valid by checking against the known hash
-        if (bodyHash) {
-          const currentBodyHash = hash(response.body);
-          if (currentBodyHash !== bodyHash) {
-            entry.up = false;
-            info.bodyHash = { expected: bodyHash, current: currentBodyHash };
-          }
-        }
+  try {
+    const response = await got[method](query, { followRedirect, headers: { cookie: "nocache=true" } });
```

Comment:
what's this `nocache=true` cookie?

Reply:
we can either use a cookie or `?nocache=true` in url to omit nginx cache and ensure that the record is served directly from siad - we need that becuase otherwise we're sometimes hitting cache and not testing end-to-end